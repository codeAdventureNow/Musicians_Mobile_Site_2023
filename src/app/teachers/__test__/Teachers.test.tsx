import Teachers from '../page';
import { render } from '@testing-library/react';

describe('Teachers', () => {
  it('should render the "Teachers" page', () => {
    const { asFragment } = render(<Teachers />);

    expect(asFragment()).toMatchSnapshot();
  });
});
