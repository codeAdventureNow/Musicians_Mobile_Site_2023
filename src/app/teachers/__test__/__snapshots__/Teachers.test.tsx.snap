// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teachers should render the "Teachers" page 1`] = `
<DocumentFragment>
  <div
    class="page"
  >
    <section
      class="heading_section"
    >
      <div
        class="container"
      >
        <h2
          class="section_heading"
        >
          Music
          <span
            class="hero_heading_highlight"
          >
             Teachers
          </span>
           + Staff
        </h2>
        <div
          class="intro_paragraph_flex"
        >
          <p
            class="intro_paragraph"
          >
            We believe that when you enjoy your instrument, you stick with it. Our patient, encouraging approach to learning blended with a friendly yet firm teaching style allows you to learn the songs you love. We strive to create an environment of self-motivated practice.
          </p>
          <p
            class="intro_paragraph"
          >
            We emphasize learning numerous musical pieces and showcasing them with peers in our exciting performance opportunities. Our teachers are employees who pass proficiency tests, multiple interviews, and a background check prior to being hired.
          </p>
        </div>
      </div>
    </section>
    <section
      class="section"
    >
      <div
        class="container"
      >
        <h2
          class="section_heading"
        >
          Staff
          <span
            class="hero_heading_highlight"
          />
        </h2>
        <div
          class="card_container"
        >
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Nate Mueller
                </h3>
                <p
                  class="teacher_role"
                >
                  Founder & Front End Engineer
                </p>
              </div>
              <img
                alt="Nate in a blue shirt"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FNate_Pic.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FNate_Pic.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FNate_Pic.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Nate Mueller is the president and founder of Musicians Mobile. He
      has been teaching music at local schools throughout the Bay Area
      since 2002. In 2003, Nate chose to take his teaching directly to
      students by providing lessons in their own homes, providing an added
      convenience for both the parents and students. Outside of teaching
      music, Nate is a successful music producer. He has provided
      entertainment for brands such as the Golden State Warriors and the
      US Olympic Team. Additionally, his compositions have been licensed
      to ABC,VH1, MTV, and featured on the hit show Modern Family.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Monica Wildfong
                </h3>
                <p
                  class="teacher_role"
                >
                  Account Administrator
                </p>
              </div>
              <img
                alt="Lady smiling"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FMWildfong.jpeg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FMWildfong.jpeg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FMWildfong.jpeg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Monica has a bachelor’s of science in accounting and worked in management, accounting, and marketing for 14 years before taking time off to raise her two children.  Monica loves traveling with her family and making custom cakes, as well as learning to play the piano.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="section"
    >
      <div
        class="container"
      >
        <h2
          class="section_heading"
        >
          Music
          <span
            class="hero_heading_highlight"
          >
             Teachers
          </span>
        </h2>
        <div
          class="card_container"
        >
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Jared W.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano Teacher
                </p>
              </div>
              <img
                alt="Jared in a suit"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FJaredW.png&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FJaredW.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FJaredW.png&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Jared teaches piano and saxophone with pop, jazz and classical influences. He is an enthusiastic piano teacher with experience performing in a variety of ensembles. Jared is a very patient instructor, offering a wealth of knowledge in technique and performance that he is passionate about sharing with his students. Additionally, Jared is well versed in organizing advanced jazz band sectionals and recitals for Buchser Middle School. Jared received his Bachelors in Music from San Jose State University.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Carl A.
                </h3>
                <p
                  class="teacher_role"
                >
                  Guitar Teacher
                </p>
              </div>
              <img
                alt="Carl smiling with a guitar"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FCarlA.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FCarlA.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FCarlA.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Carl has been delivering stellar results coaching guitar students with Musicians Mobile for 8 years. His expertise will make it easier and faster for your children to become proficient in playing the guitar whether they are beginners or advanced. As our Music Director, Carl organizes our annual Rock Band show, in which he brings more than 60 students together as a band and coaches them to perform with confidence.  Carl’s teaching style is fun, flexible, and focused. His experience allows him to teach students of varied ages, musical interests, and learning styles. He also serves as a direct mentor to our other teachers. In addition to teaching, Carl performs as a solo guitarist and with his band “Jazz With a Twist.” Carl holds a Bachelors of Arts in Music from the University of California, Santa Cruz.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Avi S.
                </h3>
                <p
                  class="teacher_role"
                >
                  Guitar, Voice, Piano Teacher
                </p>
              </div>
              <img
                alt="Avi sitting with guitar in living room"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FAviS.png&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FAviS.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FAviS.png&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Avi has been a teacher of voice and guitar in the Bay Area for more than 10 years. His patient approach and inspiring lessons help students of all ages explore a variety of interests and techniques. His lessons display his deep passion for music balanced with a focus on note reading and the development of listening skills. Avi helps students easily to understand and access even the most challenging music. Through his vocal and guitar coaching, Avi’s students learn to sing and accompany themselves on guitar with confidence. He introduces students to a diverse array of genres, including rock, pop, musical theater and classical. Avi attended the University of California at Santa Cruz, where he studied vocal performance.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Brandon L.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano Teacher
                </p>
              </div>
              <img
                alt="Brandon sitting in a living room smiling."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FBrandonL.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FBrandonL.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FBrandonL.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Brandon is a passionate pianist who has played piano for 11 years. Aside from performing in piano recitals, he has performed in choir shows, dinner parties, and rock band performances. He can teach many genres like Jazz, Pop, Video Game/Film music, and Classical, emphasizing expression and musicality. His goal is to create a safe, productive learning setting where students can truly connect with music. Brandon is instructed under Gwendolyn Mok, studying for a degree in B.M. Piano Performance at San Jose State University.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Earnest L.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano Teacher
                </p>
              </div>
              <img
                alt="Earnest sitting in a living room smiling."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FEarnestL.png&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FEarnestL.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FEarnestL.png&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Earnest began studying music and playing the piano at the age of three. He has studied music at the San Francisco Conservatory of Music, University of Pacific and San Jose State University where he earned a B.M. in Piano Performance. In his career Earnest has won numerous statewide competitions, scholarships and grants. One such competition allowed him to perform Tchaikovsky’s Piano Concerto No. 1 with the Stockton Symphony. Earnest is currently pursuing a Masters degree under the mentorship of Dr. Gwendolyn Mok then will continue to work on a doctorate degree, with future plans to teach at a college/university. Earnest believes in smart practice habits to maximize time and effort with a lesson plan that engages the student in a fun and interesting way. In addition to classical music, Earnest can play and teach R&B, pop, top 40, Broadway and Disney songs.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Joshua C.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano Teacher
                </p>
              </div>
              <img
                alt="Joshua sitting in a living room smiling."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FJoshC.jpeg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FJoshC.jpeg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FJoshC.jpeg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Joshua started playing piano at the age of 6 and took piano lessons for 8 years. Since then, he’s been a self-taught performer/composer who played in several orchestras, weddings, and public gigs. He is experienced in teaching students funk, gospel, classical, and video game/movie music. However, he is most specialized in teaching ear-playing, improvisation, music production, and composition theory. He highly believes that the student’s goals come first and will work his lessons around what his students want to achieve. He is also able to arrange custom sheet music for students, even for songs that have no existing piano sheets available. Joshua is currently pursuing a bachelor’s degree in Economics, with a minor in music and computer science at San Jose State University. 
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Jillian V.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano Teacher
                </p>
              </div>
              <img
                alt="Jillian standing next to a window smiling."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FJillianV.jpeg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FJillianV.jpeg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FJillianV.jpeg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Jillian has played piano since the age of six and completed a Bachelor of Arts in Music at the University of Puget Sound in 2018. Since then, she has been a passionate teacher and musician. As a teacher, she believes that students learn best by pursuing their own goals and motivations. She is experienced in teaching Classical and Popular music. Her objective is to create a comfortable environment in which students can grow and connect with music. As a result, she is always striving to find better ways to adapt to different learning methods in order to engage with students. 
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Alex DL.
                </h3>
                <p
                  class="teacher_role"
                >
                  Drum Teacher
                </p>
              </div>
              <img
                alt="Alex sitting on steps holding drum sticks."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FAlexDL.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FAlexDL.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FAlexDL.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Alex has centered his life around drums. He got his first pair of drumsticks at five years old and practiced constantly until he got a drum kit at twelve years old.  He immediately began performing as much as possible, from the church choir and high school drumline as a young person to now. Alex has played in bands around the Bay Area since 2010. In 2016, he lived in Vietnam, where he taught English and played in a metal band. Alex now plays professionally in multiple rock and folk bands.  He especially enjoys working with young people and sharing his love of music. He is a passionate, engaging, and caring teacher who looks forward to helping aspiring musicians discover their drumming potential.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Xavier C.
                </h3>
                <p
                  class="teacher_role"
                >
                  Drum Teacher
                </p>
              </div>
              <img
                alt="Xavier sitting outside of house smiling."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FXavierC.png&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FXavierC.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2FXavierC.png&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Xavier is a passionate and accomplished percussion teacher with over 11 years behind the drum kit, and 5 years as an instructor. Xavier was recognized as a talented musician as early as high school, where he received multiple awards and recognition for his skills on drum set and concert percussion. Upon graduating, his high school hired Xavier to lead the marching band drum-line and percussion ensemble. Under his guidance, his drum-line ranked in the top 10 percussion ensembles in the world. Apart from teaching, Xavier actively performs as a drummer in various jazz groups throughout the Bay Area.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Heidi M.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano, Voice, Violin Teacher
                </p>
              </div>
              <img
                alt="Heidi sitting in living room smiling."
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FHeidiM.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FHeidiM.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FHeidiM.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Heidi has enjoyed a lifetime of playing music in both America and Europe, and has over 30 years of teaching experience. Throughout her accomplished performing career, Heidi has played in many major concert halls in America, Europe, and Asia, with artists like Pavarotti, Yo-Yo Ma, and Jesse Norman. She has also played on movie soundtracks like “You’ve Got Mail” and in string sections for artists like Ray Charles, Aretha Franklin, and Burt Bacharach. Recently, Heidi was a featured soloist with the San Francisco Girls Chorus. In addition, she has conducted youth orchestras, choirs and other chamber music ensembles for over 15 years. Heidi is a graduate of the Oberlin Conservatory, The Royal Conservatory of the Hague, and The Juilliard School.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Marcus H.
                </h3>
                <p
                  class="teacher_role"
                >
                  Piano, Guitar Teacher
                </p>
              </div>
              <img
                alt="Marcus smiling"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FMarcusH.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FMarcusH.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FMarcusH.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Marcus studied under Dr. Stephen Thomas at Brigham Young University, earned a Piano Performance degree from UC Stanislaus with Honors.  He has taught piano for 15 years and is a current teacher at the Challenger School.   Marcus also knows how to play guitar and can teach beginners.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Bryan C.
                </h3>
                <p
                  class="teacher_role"
                >
                  Guitar, Piano Teacher
                </p>
              </div>
              <img
                alt="Bryan in music studio with guitar"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FBryanC.jpeg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FBryanC.jpeg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FBryanC.jpeg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Bryan is a Bay Area musician with over a decade of experience playing guitar and keyboard instruments. His notable performances include opening for renowned acts such as Y&T and Michael Franti. Bryan has performed for corporate entities like Facebook, SanDisk, and the San Jose Earthquakes. Bryan has a true passion for music and its ability to bring focus, determination, and purpose into people’s lives. He loves to teach problem-solving, creativity, and improvisation through music. He is mostly self-taught, but began formal training in Jazz Studies and Classical Guitar in 2015 at De Anza College. Bryan teaches many styles with an emphasis on rock, blues, soul, pop, and jazz music.
              </p>
            </div>
          </div>
          <div
            class="card"
          >
            <div
              class="cardTop"
            >
              <div>
                <h3
                  class="teacher_name"
                >
                  Daniil M.
                </h3>
                <p
                  class="teacher_role"
                >
                  Guitar Teacher
                </p>
              </div>
              <img
                alt="Daniil in music studio with guitar"
                class="image"
                data-nimg="1"
                decoding="async"
                height="150"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2FDaniilM.jpg&w=384&q=75"
                srcset="/_next/image?url=%2Fimages%2FDaniilM.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2FDaniilM.jpg&w=384&q=75 2x"
                style="color: transparent; max-width: 100%; height: auto;"
                width="150"
              />
            </div>
            <div
              class="cardBottom"
            >
              <p>
                Daniil is majoring in Music Production at Berklee College of Music.  He is interested in sharing his knowledge and experience with those passionately interested in learning guitar and music production.  Daniil began his music journey at a guitar academy in South Korea at the age of eight, studying classical guitar.  After completing a six year course he picked up acoustic and electric guitar.  Since then, Daniil has been performing as a guitarist in various church worship teams and providing guitar lessons in countries South Korea, Uzbekistan, and the United States.  Daniil’s lessons focus on developing fingerstyle techniques, learning anime, film, video game songs, and producing high-quality music with the help of technology and music production techniques.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
